modules/installation-special-config-storage.adoc

```diff
@@ -46,72 +79,82 @@ Mirroring is available only for user-provisioned infrastructure deployments on {
 ====
 
 [id="installation-special-config-storage-procedure_{context}"]
-== Configuring disk encryption or mirroring
+== Configuring disk encryption and mirroring
 
-.Procedure
+You can enable and configure encryption and mirroring during an {product-title} installation.
 
-To enable boot disk encryption or mirroring during {product-title} deployment:
+.Prerequisites
+
+* You downloaded the {product-title} installation program on your installation node.
+* You installed Butane on your installation node.
++
+[NOTE]
+====
+Machine configs are used to configure the control plane and compute machines. In some situations, such as boot disk mirroring, you can use Butane to create the machine configs for you, making node configuration easier. See the _Creating machine configs with Butane_ section for more information.
+====
++
+* You have access to a {op-system-base-full} 8 system that can be used to generate a thumbprint of the Tang exchange key.
+
+.Procedure
 
 . If you want to use TPM v2 to encrypt your cluster, check to see if TPM v2 encryption needs to be enabled in the BIOS on each node. This is required on most Dell systems. Check the manual for your computer.
 
 . If you want to use Tang to encrypt your cluster, follow these preparatory steps:
 
-.. Access a Red Hat Enterprise Linux server from which you can configure the encryption settings.
+.. Set up a Tang server or access an existing one. See link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/security_hardening/configuring-automated-unlocking-of-encrypted-volumes-using-policy-based-decryption_security-hardening#network-bound-disk-encryption_configuring-automated-unlocking-of-encrypted-volumes-using-policy-based-decryption[Network-bound disk encryption] for instructions. See link:https://youtu.be/2uLKvB8Z5D0[Securing Automated Decryption New Cryptography and Techniques] for a presentation on Tang.
```

Comment:
Do you have/need internal permission to use this YouTube video?

Reply:
From what I can see, that URL was added [as part of PR 19144](https://github.com/openshift/openshift-docs/pull/19144/files#diff-4543a60a972c20c29aa95cfc1fc230f12412be5185ae59a719a487df8f99503dR14) in `modules/installation-special-config-encrypt-disk-tang.adoc`. In [PR 33337](https://github.com/openshift/openshift-docs/pull/33337/files#diff-d2ca5b8fa954c04153c54b7e790fd5834f7838bfaffeaab7243206b2430faf4bR57) that content was moved to `modules/installation-special-config-storage.adoc`.

From a quick search in PR 19144, I can't see any specific approval request for the link. @vikram-redhat do we need to request approval for this URL which already exists in the docs? Thanks.

Reply:
Following a conversation with @vikram-redhat, I am removing the YouTube link.